src/pages/modals/create-avatar.tsx

```diff
@@ -0,0 +1,113 @@
+// Store
+import { useRef, useState } from 'preact/hooks'
+import { useStore } from '../../store'
+import cancel from '../../assets/imgs/cancel.svg?url'
+import checkMarkBlue from '../../assets/imgs/checkMarkBlue.svg?url'
+import iconRotate from '../../assets/imgs/iconRotate.svg?url'
+import { blobToDataURL } from '../../lib/canvas'
+import { Cropper, CropperRef } from '../../components/cropper'
+import { JSXInternal } from 'preact/src/jsx'
+
+interface Props {
+	children: JSX.Element | JSX.Element[]
+}
+
+export const CreateAvatar = ({ children }: Props) => {
+	const [avatar, setAvatar] = useState<string>('')
```

Comment:
Not sure it makes much sense, but I guess we should either put the default avatar back, or rework the screen slightly to force the user to choose a picture instead of displaying an empty `<Cropper>`.

Reply:
Good point and I think the UX should be different. IMHO when user clicks on the avatar without having any, they should right away be asked to choose a file or take picture. Then we can always have cropper with something in it and there is one less click needed from the user.